package-lock.json

```diff
@@ -5,7 +5,6 @@
     "requires": true,
     "packages": {
         "": {
-            "name": "aws-toolkit-vscode",
```

Comment:
?

Reply:
Older version of `npm` probably.

Reply:
Possibly, want me to try and update before merge?